presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergMetadataListing.java

```diff
@@ -117,6 +117,6 @@ public void testTableDescribing()
     public void testTableValidation()
     {
         assertQuerySucceeds("SELECT * FROM iceberg.test_schema.iceberg_table1");
-        assertQueryFails("SELECT * FROM iceberg.test_schema.hive_table", "Not an Iceberg table: test_schema.hive_table");
+        assertQueryFails("SELECT * FROM iceberg.test_schema.hive_table", ".*Not an Iceberg table: test_schema.hive_table");
```

Comment:
why did the error regex change?

Reply:
I'd recommend adding the new error message instead of .* - so its clearer.

Reply:
With preprocessing enabled(its disabled by default at the moment), the error message also have the exception class name, hence adding regex.